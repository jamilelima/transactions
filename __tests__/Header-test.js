import React from 'react';
import {shallow} from 'enzyme';
import Header from '../src/components/header';

describe('Header', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(
        <Header label="test label" amountValue={200} />,
      );
      expect(component).toMatchSnapshot();
    });
  });
});
