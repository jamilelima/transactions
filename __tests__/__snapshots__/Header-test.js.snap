// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Rendering should match to snapshot 1`] = `
<Styled(View)>
  <View>
    <Styled(Text)>
      Segunda-feira
    </Styled(Text)>
    <Styled(Text)>
      20/04/2020
    </Styled(Text)>
  </View>
  <View>
    <Styled(Text)>
      Seu saldo
    </Styled(Text)>
    <Styled(Text)>
      R$ 
      200.00
    </Styled(Text)>
  </View>
</Styled(View)>
`;
